lib/phoenix/socket/transport.ex

```diff
@@ -201,10 +202,11 @@ defmodule Phoenix.Socket.Transport do
     if channel = socket.handler.__channel__(topic, socket.transport_name) do
       socket = %Socket{socket | topic: topic, channel: channel}
 
+      filtered_payload = Instrument.filter_values(msg.payload, Application.get_env(:phoenix, :filter_parameters))
```

Comment:
Should we bake this into a module attribute? We can document recompile instructions and it's something that shouldn't change often.